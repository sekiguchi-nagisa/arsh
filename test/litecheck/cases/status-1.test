#!/bin/sh
# RUN: /bin/sh $self
exit 100
# STATUS: 120