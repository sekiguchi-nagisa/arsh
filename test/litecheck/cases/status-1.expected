exit status is mismatched, expected: `120', actual: `100'